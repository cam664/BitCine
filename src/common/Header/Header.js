import React from 'react';

const Header = () => {
  return (
    <header>
      This is the header.
    </header>
  );
}

export default Header;

