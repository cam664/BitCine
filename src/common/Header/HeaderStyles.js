export default {
  header: `
    box-sizing: border-box;
    width: 100%;
    height: 70px;
    padding: 10px 2.5%;
    position: fixed;
    top: 0;
    left: 0;
    background-color: #fff;
    box-shadow: 0 1px 1px rgba(0,0,0,0.1);
    display: flex;
    align-items: center;
    justify-content: center;
    img {
      width: 100px;
    }
  `
}