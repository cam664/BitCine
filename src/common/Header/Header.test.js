import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';
import { BrowserRouter as Router  } from 'react-router-dom';

describe('Header', () => {
  it('renders a snapshot', () => {
    const component = renderer.create(
      <Router>
        <Header />
      </Router>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});