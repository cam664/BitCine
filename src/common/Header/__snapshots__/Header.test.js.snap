// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders a snapshot 1`] = `
<header
  className="css-j2ge83-Header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      src="logo-star-wars.png"
    />
  </a>
</header>
`;
